addons/payment_ogone/models/payment_acquirer.py

```diff
@@ -43,6 +43,19 @@ def _get_validation_amount(self):
 
         return 1.0
 
+    def _get_redirect_form_view(self, is_validation=False):
+        """ Override of payment to return the FlexCheckout form for validation operations.
+
+        :param bool is_validation: Whether the operation is a validation
+        :return: The redirect form template
+        :rtype: record of `ir.ui.view`
+        """
+        res = super()._get_redirect_form_view()
+        if self.provider != 'ogone' or not is_validation:
+            return res  # self.redirect_form_view_id
+
+        return self.env.ref('payment_ogone.redirect_form_validation')
```

Comment:
Let's give the final word to TBE. I don't really consider saas-14.3 as a stable (from a DB PoV) since there is only one DB running it and its manager is reviewer of this PR :p

Reply:
Hello :)

14.3 is running only on our instance, we are not yet ready to roll out this version on SaaS. This change was asked knowing this fact. But you are right to ask this question :)